db/cache/cache.go

```diff
@@ -0,0 +1,139 @@
+package cache
+
+import (
+	"reflect"
+	"sync"
+	"time"
+
+	"github.com/livepeer/leaderboard-serverless/common"
+	"github.com/livepeer/leaderboard-serverless/models"
+)
+
+type Cache interface {
```

Comment:
Do we need this interface?

Reply:
I would suggest leaving it.  It is just a harmless abstraction to support swapping out the implementation in the future and increase testability.